src/main/thrift/parquet.thrift

```diff
@@ -309,6 +309,13 @@ struct Statistics {
    7: optional bool is_max_value_exact;
    /** If true, min_value is the actual minimum value for a column */
    8: optional bool is_min_value_exact;
+   /** 
+    * count of NaN values in the column; only present if physical type is FLOAT
+    * or DOUBLE, or logical type is FLOAT16. 
+    * Readers MUST distinguish between nan_count not being present and nan_count == 0.
+    * If nan_count is not present, readers MUST NOT assume nan_count == 0.
+    */
+   9: optional i64 nan_count;
```

Comment:
I would find this easier to understand if it were more concise and didn't have a double negative. Here is a suggestion:

```suggestion
   /** 
    * count of NaN values in the column; only present if physical type is FLOAT
    * or DOUBLE, or logical type is FLOAT16. 
    * If this field is not present, readers must assume NaNs may be present 
    * (MUST NOT assume nan_count == 0).
    */
   9: optional i64 nan_count;
```